src/index.html

```diff
@@ -11,7 +11,177 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
-    <script type="text/javascript" src="./main.js"></script>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <fieldset class="fieldset">
+        <legend>
+        Personal information
+        </legend>
+        <div>
+          <label for="Surname">
+            Surname:
+          </label>
+          <input id="Surname" type="text"
+          name="Surname"
```

Comment:
<img width="966" alt="Screenshot 2022-11-11 at 10 33 35" src="https://user-images.githubusercontent.com/104769168/201299412-f3667f2c-5f5a-453e-8d5b-5d579b551b91.png">
```suggestion
          name="surname"
```
fix it all over the page